### DISCUSSION TITLE
nchat suddenly display empty window, doesn't seem to connect to Whatsapp anymore

### DISCUSSION THREAD

**Discussion by 82MHz** [d99kris/nchat#247] [Q&A]
I run nchat on two different machines, both archlinux, both run nchat version v4.53. 

I noticed that on both of them nchat now only starts up with an empty window and doesn't seem to connect to Whatsapp anymore, see screenshot. 

In Whatsapp if I look at my connected devices, I see that they last connected some time yesterday. So nchat/whatsmeow doesn't seem to be connecting to the Whatsapp servers anymore.

Any idea what might have caused this, and how I could restart the connection?

![Screenshot_20240604_081923](https://github.com/d99kris/nchat/assets/16221316/1aec6544-e05b-4850-918d-db17956efbdf)

**Comment by d99kris** [comment]
Hi @82MHz - thanks for reporting the issue. This issue should be resolved with dbc056e (whatsmeow has been updated), it's available in the latest nchat v4.86 - just released https://github.com/d99kris/nchat/releases/tag/v4.86 

Let me know if you still encounter any issues after updating to latest nchat. Thanks!

**Comment by 82MHz** [comment]
Works perfectly, thanks for updating! :)



Am 04.06.24 um 13:20 schrieb Kristofer Berggren:

**Comment by 82MHz** [comment]
Also, I just want to say - nchat has become one of my favourite pieces 
of software in recent months, I'm not lying when I say I use it every 
single day :)

So thank you so much for developing it and continuing to work on it!



Am 04.06.24 um 13:20 schrieb Kristofer Berggren:

**Reply by d99kris** [reply]
Thanks for those nice words 👍
